### PR TITLE
[docker-restapi]: add restapi config to config Db

### PR DESCRIPTION
**Pull request by sumukhatv**
**- Why I did it**
Add RESTAPI config to Config DB
**- How I did it**
Through the minigraph.py script
**- How to verify it**
using redis-cli
**- Description for the changelog**

Add RESTAPI config to Config DB

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: src/sonic-config-engine/minigraph.py [hunk 1]
```diff
@@ -836,7 +836,19 @@ def parse_xml(filename, platform=None, port_config_file=None):
...
-
+    results['RESTAPI'] = {
+        'config': {
+            'client_auth': 'true',
+            'allow_insecure': 'false',
+            'log_level': 'trace'
+        },
+        'certs': {
+            'server_crt': '/etc/sonic/certificates/restapiserver.crt',
+            'server_key': '/etc/sonic/certificates/restapiserver.key',
```
Comment (lguohan) [inline src/sonic-config-engine/minigraph.py:847]: credentials

Comment (sumukhatv) [reply src/sonic-config-engine/minigraph.py]: changed

**Comment by sumukhatv** [comment]
retest broadcom please

**Comment by sumukhatv** [comment]
retest vs please

**Comment by lguohan** [comment]
vs test failure is fixed in #4499

**Pull request merged**